services/blockchain-connector/shared/sdk/blocks.js

```diff
@@ -24,31 +24,31 @@ const {
 
 const getLastBlockDecoded = async () => {
 	const block = await getLastBlock();
-	const decodedBlock = decodeBlock(block);
+	const decodedBlock = formatBlock(block);
 	return decodedBlock;
 };
 
 const getBlockByHeightDecoded = async (height) => {
 	const block = await getBlockByHeight(height);
-	const decodedBlock = decodeBlock(block);
+	const decodedBlock = formatBlock(block);
 	return decodedBlock;
 };
 
 const getBlocksByHeightBetweenDecoded = async ({ from, to }) => {
 	const blocks = await getBlocksByHeightBetween({ from, to });
-	const decodedBlocks = blocks.map(b => decodeBlock(b));
+	const decodedBlocks = blocks.map(b => formatBlock(b));
 	return decodedBlocks;
 };
 
 const getBlockByIDDecoded = async (id) => {
```

Comment:
We probably do not need this method anymore. Could you please refactor the code to use `getBlockByID` instead?
Both of them should respond the same now.